src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Internal/MsQuicApi.cs

```diff
@@ -9,6 +9,8 @@ namespace System.Net.Quic.Implementations.MsQuic.Internal
 {
     internal unsafe sealed class MsQuicApi
     {
+        private static readonly Version MinWindowsVersion = new Version(10, 0, 20145, 1000);
```

Comment:
Not much worth in a field since it is only initialized and accessed once. A property is fine.

```suggestion
        private static Version MinWindowsVersion => new Version(10, 0, 20145, 1000);
```

Reply:
It's actually accessed 4 times on check and 5th time on logging...